CMakeLists.txt

```diff
@@ -21,19 +21,52 @@ if (NOT CMAKE_BUILD_TYPE)
             "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif ()
 
+# Set build options
+option (BUILD_UFS "Build GOCART component for UFS" OFF)
+
 set (DOING_GEOS5 YES)
 
 # Should find a better place for this - used in Chem component
 set (ACG_FLAGS -v)
 set (FV_PRECISION R4R8)
 
+# Ensure we build as 32-bit
+message ("Force 32-bit build for GOCART")
```

Comment:
@rmontuoro We don't really want people by fiat changing flags for GEOS. Is this something you can protect with `if (BUILD_UFS)`? Otherwise, it probably won't be accepted.

Reply:
Ditto.

Reply:
Agreed.

Reply:
Done. See commit d06f07d